main.go

```diff
@@ -31,12 +31,22 @@ type config struct {
 	ManualMerge      bool   `env:"manual_merge,opt[yes,no]"`
 }
 
-func printLogAndExportEnv(gitCmd git.Git, format, env string) error {
+const (
+	maxCommitBodyLenght = 72
+)
+
+func printLogAndExportEnv(gitCmd git.Git, format, env string, trim bool) error {
 	l, err := output(gitCmd.Log(format))
 	if err != nil {
 		return err
 	}
 
+	if trim && len(l) > maxCommitBodyLenght {
+		trimmedValue := l[:maxCommitBodyLenght]
```

Comment:
Would've liked to append an indicator that it was trimmed, like `…`

Reply:
Great idea, added indicator.